packages/amplication-data-service-generator-runner/scripts/create-input-json.ts

```diff
@@ -0,0 +1,26 @@
+import { EnumResourceType } from "@amplication/data-service-generator/models";
+import { appInfo } from "@amplication/data-service-generator/tests/appInfo";
+import { plugins } from "@amplication/data-service-generator/tests/constants/example-plugins";
+import entities from "@amplication/data-service-generator/tests/entities";
+import roles from "@amplication/data-service-generator/tests/roles";
+import { writeFileSync } from "fs";
+import { join } from "path";
+import { format } from "prettier";
+
+if (require.main === module) {
+  createInputJsonFile();
+}
+
+function createInputJsonFile() {
+  const object = {
+    entities,
+    roles,
+    resourceInfo: appInfo,
+    resourceType: EnumResourceType.Service,
+    pluginInstallations: [plugins.kafka, plugins.postgresPlugin],
+  };
+  const fileName = "input.json";
```

Comment:
Why input is hardcoded here? It supposed to be configurable, path in file system may change.

Reply:
As far as I see this script is for development purposes only, it creates input json from test entities, roles and etc. Am I right?

Reply:
Yes, you are right and this is also the answer to your question.